guides/common/modules/proc_chainbooting-ipxe-from-pxelinux.adoc

```diff
@@ -1,53 +1,51 @@
 [id="Chainbooting_iPXE_from_PXELinux_{context}"]
 = Chainbooting iPXE from PXELinux
 
-Use this procedure to set up iPXE to use a built-in driver for network communication or UNDI interface.
-To use HTTP with iPXE, use iPXE build with built-in drivers (`ipxe.lkrn`).
-Universal Network Device Interface (UNDI) is a minimalistic UDP/IP stack that implements TFTP client, however, cannot support other protocols like HTTP (`undionly-ipxe.0`).
-You can choose to either load `ipxe.lkrn` or `undionly-ipxe.0` file depending on the networking hardware capabilities and iPXE driver availability.
+You can set up iPXE to use a built-in driver for network communication (`ipxe.lkrn`) or UNDI interface (`undionly-ipxe.0`).
+You can choose to load either file depending on the networking hardware capabilities and iPXE driver availability.
 
-.Chainbooting iPXE directly or with UNDI workflow
-. Host powers on
-. PXE driver retrieves the network credentials using DHCP
-. PXE driver retrieves the PXELinux firmware `pxelinux.0` using TFTP
-. PXELinux searches for the configuration file on the TFTP server
-. PXELinux chainloads iPXE `ipxe.lkrn` or `undionly-ipxe.0`
-. iPXE retrieves the network credentials, including an HTTP URL, using DHCP again
-. iPXE chainloads the iPXE template from the template {SmartProxy}
-. iPXE loads the kernel and initial RAM disk of the installer
+Universal Network Device Interface (UNDI) is a minimalistic UDP/IP stack that implements TFTP client, however, cannot support other protocols like HTTP.
+To use HTTP with iPXE, use the iPXE build with built-in drivers (`ipxe.lkrn`).
 
-.Prerequisites
-include::snip_prerequisite-configured-smart-proxy-registration-provisioning.adoc[]
+.Chainbooting iPXE workflow
+. Host powers on.
+. PXE driver retrieves the network credentials by using DHCP.
+. PXE driver retrieves the PXELinux firmware `pxelinux.0` by using TFTP.
+. PXELinux searches for the configuration file on the TFTP server.
+. PXELinux chainloads iPXE `ipxe.lkrn` or `undionly-ipxe.0`.
+. iPXE retrieves the network credentials, including an HTTP URL, by using DHCP again.
+. iPXE chainloads the iPXE template from Templates {SmartProxy}.
+. iPXE loads the kernel and initial RAM disk of the installer.
 
-.Configuring {ProjectServer} to use iPXE
-You can use the default template to configure iPXE booting for hosts.
-If you want to change the default values in the template, clone the template and edit the clone.
+.Configuring the templates
+You can use the default templates as described below to configure iPXE booting for hosts.
+If you require modifications of a default template, clone the template, edit the clone, and associate the cloned template instead of the default template.
+For more information, see xref:cloning-provisioning-templates_{context}[].
 
 .Procedure
```

Comment:
Thank you for your feedback. I'm not sure if it's such a good idea, because then it would be difficult to bind the workflow of those 3 procedures together. For now, I'm keeping it as a single procedure. However, I'll consider it during the Prov. guide revamp.

Reply:
Another possibility would be to turn those 3 processes into sub-procedures. That would also reduce the number of steps and make it easier to scan.

Reply:
I'll just randomly chime in to support the idea of breaking the long procedure into smaller pieces. 22 steps is too much for a single procedure. Splitting it into meaningful chunks would give readers a chance to take a breath during the process.

I think I can see 4 meaningful chunks like that -- the first 3 ending after each "Submit" step.

Reply:
While I don't completely agree with the idea of those many chunks but if it would be me, 
* Points 1 - 18 are relevant to the topic header i.e. `Configuring templates for provisioning`  and should remain there
* Points 19 - 22 , can become a different segment called `Configuring Host Groups for Provisioning` 

Too many sections are not always good and can become confusing at times.

Reply:
Points taken. However, in such case I'd go with 3rd-level sections.